Makefile

```diff
@@ -38,11 +38,16 @@ test-e2e: ginkgo ## Run the e2e tests
 verify: tidy generate ## Verify the current code generation and lint
 	git diff --exit-code
 
-install: generate
-	# TODO(tflannag): Introduce registryv1 manifests
+install-k8s: generate
 	kubectl apply -f manifests
 	kubectl apply -f provisioner/k8s/manifests
 
+install-registryv1: generate
+	kubectl apply -f manifests
+	# TODO create deployment manifests for registryv1 provisioner
+
+install: install-k8s install-registryv1
```

Comment:
This ends up applying the manifests twice, but since there should not be any changes it's basically a no-op.